config.js

```diff
@@ -1252,6 +1252,10 @@ var config = {
     // Prevent the filmstrip from autohiding when screen width is under a certain threshold
     // disableFilmstripAutohiding: false,
 
+    // Disables user resizable filmstrip. Also, allows configuration of the filmstrip (width, tiles aspect ratios)
+    // through the interfaceConfig options.
+    // disableResizableFilmstrip: false,
```

Comment:
Can you please create a `filmstrip: { ... }` section instead? This way we can start to migrate those settings in interface_config.js there after this PR lands, instead of having a dozen of scattered around settings.